src/gensxg.c

```diff
@@ -489,6 +489,11 @@ void write_sxg(const sxg_signer_list_t* signers,
     exit(EXIT_FAILURE);
   }
 
+  if (fclose(out) != 0) {
```

Comment:
I'm not sure if this is good when `out` is `stdout`.

Reply:
Actually it's not good if `out == stdout` since this indeed closes `stdout` and further writes to `stdout` including `printf` and so on will be undefined behaviors.

Reply:
We shouldn't write anything other than sxg when ```-o -``` is specified. Writing something further to stdout itself is a bug, I think.

Reply:
At the same time, it is not generally recommended to fclose(stdout) anyway (https://stackoverflow.com/questions/4972994)

Reply:
Confirmed. IMHO it generally doesn't sound very good to close stdout although it's not truly required (also for changes in the future), but that's also true that we have a justification for now. Just adding some comment? Anyway I'd defer to @kumagi.

Reply:
Thank you for discussion, I'll leave stdout open.